MediaBrowser.Controller/MediaEncoding/EncodingHelper.cs

```diff
@@ -424,7 +424,12 @@ public string GetAudioEncoder(EncodingJobInfo state)
 
             if (string.Equals(codec, "aac", StringComparison.OrdinalIgnoreCase))
             {
-                return "aac -strict experimental";
+                // Use libfdk_aac for better audio quality if using custom build of FFmpeg which has fdk_aac support
+                if (_mediaEncoder.SupportsEncoder("libfdk_aac"))
+                {
+                    return "libfdk_aac";
+                }
```

Comment:
```suggestion
                }

```
Empty line after closing braces